.github/workflows/ci.yml

```
@@ -3,15 +3,23 @@ on:
   workflow_dispatch:
   pull_request:
 jobs:
-  validate:
+  CI:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [12.x]
+    permissions:
+      id-token: write
+      contents: read
     steps:
-      - uses: actions/checkout@v2
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
+      - uses: actions/checkout@v3
+      - name: Setup node
+        uses: actions/setup-node@v3.7.0
         with:
-          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
       - run: ./script/ci
+      - uses: actions/upload-artifact@v3
+        with:
+          path: dist
+      - uses: guardian/actions-static-site@v2
+        with:
+          app: editions-card-builder
+          domain: editions-card-builder.gutools.co.uk
```

test by deploying to this or some other alternative name first, then switching later? to avoid bringing down the real thing which is still accessible by some ISPs. hopefully the site doesn't hardcode its domain anywhere

```suggestion
          domain: editions-card-builder-2.gutools.co.uk
```

---

> test by deploying to this or some other alternative name first, then switching later? to avoid bringing down the real thing which is still accessible by some ISPs. hopefully the site doesn't hardcode its domain anywhere

I thought about this, but that leaves behind detritus in the deployTools account, plus DevX will need to add the domain to the google account and remove after. If somebody from @guardian/digital-cms would like to please review, and they also don't want to take the risk then I'm happy to do Andrew's suggestion and pair with DevX to cleanup after.